src/Libraries/Microsoft.Extensions.Diagnostics.ResourceMonitoring/Calculator.cs

```diff
@@ -38,8 +38,7 @@ public static ResourceUtilization CalculateUtilization(in Snapshot first, in Sna
         long newUsageTicks = second.KernelTimeSinceStart.Ticks + second.UserTimeSinceStart.Ticks;
         long totalUsageTickDelta = newUsageTicks - oldUsageTicks;
 
-        var utilization = Math.Max(0.0, totalUsageTickDelta / totalSystemTicks * Hundred);
-        var cpuUtilization = Math.Min(Hundred, utilization);
+        var cpuUtilization = Math.Max(0.0, totalUsageTickDelta / totalSystemTicks * Hundred);
```

Comment:
Please use the explicit type instead of var since the type is not obvious (per the [style guide](https://github.com/dotnet/extensions/blob/main/CONTRIBUTING.md#dos-and-donts), point 10).
I know it's an original code, but since we're touching it we should have it fixed.

Reply:
Style fixed